test/integration/framework/src/siftool/cosmos.py

```diff
@@ -19,10 +19,20 @@ def balance_normalize(bal: CompatBalance = None) -> Balance:
     return {k: v for k, v in bal.items() if v != 0}
 
 
-def balance_add(bal1: Balance, bal2: Balance) -> Balance:
+def balance_add(*bal: Balance) -> Balance:
```

Comment:
This function name is confusing to me, I think for a while and then realize what is doing. I think balance_aggregate or balance_combine is better to describe its functionality.